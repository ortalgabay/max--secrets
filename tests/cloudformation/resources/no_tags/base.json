{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Resources": {
    "S3Bucket": {
      "Type": "AWS::S3::Bucket",
      "Properties": {
        "Tags": [
          {
            "Key": "yor_trace",
            "Value": "15f72e62-1a8e-471a-ae4c-ed6d19fe4071"
          }
        ],
        "AccessControl": "PublicRead",
        "WebsiteConfiguration": {
          "IndexDocument": "index.html",
          "ErrorDocument": "error.html"
        }
      },
      "DeletionPolicy": "Retain"
    }
  }
}