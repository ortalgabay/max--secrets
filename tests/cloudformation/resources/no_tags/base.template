AWSTemplateFormatVersion: 2010-09-09
Resources:
  S3Bucket:
    Type: AWS::S3::Bucket
    Properties:
      AccessControl: PublicRead
      WebsiteConfiguration:
        IndexDocument: index.html
        ErrorDocument: error.html
      Tags:
        - Key: yor_trace
          Value: 139e3ee0-85fa-42af-89af-65abd013b7f2
    DeletionPolicy: Retain