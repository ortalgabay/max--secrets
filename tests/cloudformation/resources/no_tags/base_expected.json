{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Resources": {
    "S3Bucket": {
      "Type": "AWS::S3::Bucket",
      "Properties": {
        "Tags": [
          {
            "Key": "new_tag",
            "Value": "new_value"
          },
          {
            "Key": "another_tag",
            "Value": "another_val"
          },
          {
            "Key": "yor_trace",
            "Value": "670a0c92-0e5b-48b3-926c-513ade6ea097"
          }
        ],
        "AccessControl": "PublicRead",
        "WebsiteConfiguration": {
          "IndexDocument": "index.html",
          "ErrorDocument": "error.html"
        }
      },
      "DeletionPolicy": "Retain"
    }
  }
}